WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductCoordinator.swift

```diff
@@ -55,6 +55,16 @@ final class AddProductCoordinator: Coordinator {
     private var addProductWithAIEligibilityChecker: ProductCreationAIEligibilityCheckerProtocol
     private var addProductWithAIBottomSheetPresenter: BottomSheetPresenter?
 
+    private lazy var productCreationAISurveyPresenter: BottomSheetPresenter = {
+        BottomSheetPresenter(configure: { bottomSheet in
+            var sheet = bottomSheet
+            sheet.prefersEdgeAttachedInCompactHeight = true
+            sheet.largestUndimmedDetentIdentifier = .none
```

Comment:
Please remove this line, as it would disable the dimmed background of the bottom sheet, making it possible to interact with the views in the background.

Reply:
Removed in https://github.com/woocommerce/woocommerce-ios/pull/11390/commits/1fe99560b1fdb4ed9bf73be8b71b24d55fa1a330